rules/aws/persistence_rds_instance_creation.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2021/06/06"
+maturity = "production"
+updated_date = "2021/06/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of an Amazon Relational Database Service (RDS) Aurora database instance.
+"""
+false_positives = [
+    """
+    Valid Instances may be created by a system or network administrator. Verify whether the user identity, user agent,
+    and/or hostname should be making changes in your environment. Instances creations from unfamiliar users or hosts
+    should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
```

Comment:
```suggestion
    A database instance may be created by a system or network administrator. Verify whether the user identity, user agent,
    and/or hostname should be making changes in your environment. Instances creations from unfamiliar users or hosts
    should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
```